### ISSUE TITLE
Navigation: Fix missing top border on focus ring in submenu item

### ISSUE THREAD

**Issue by axelavargas** [grafana/grafana#42750]
**What happened**:

This ticket needs to be implemented after this [PR](https://github.com/grafana/grafana/pull/41618) is merged.

When navigating through the keyboard, the first element of the submenu does not have the top border. 

![OffsetBug](https://user-images.githubusercontent.com/239999/144628568-b05a4f20-bae5-4573-9272-38ef71a34e29.png)
